guides/source/configuring.md

```diff
@@ -375,6 +375,28 @@ The MySQL adapter adds one additional configuration option:
 
 * `ActiveRecord::ConnectionAdapters::Mysql2Adapter.emulate_booleans` controls whether Active Record will consider all `tinyint(1)` columns as booleans. Defaults to `true`.
 
+The SQLite3Adapter adapter adds one additional configuration option:
+
+* `ActiveRecord::ConnectionAdapters::SQLite3Adapter.represent_boolean_as_integer`
+indicates whether boolean values are stored in sqlite3 databases as 1 and 0 or
+'t' and 'f'. Leaving `ActiveRecord::ConnectionAdapters::SQLite3Adapter.represent_boolean_as_integer`
+set to false is deprecated. SQLite databases have used 't' and 'f' to serialize
+boolean values and must have old data converted to 1 and 0 (its native boolean
+serialization) before setting this flag to true. Conversion can be accomplished
+by setting up a rake task which runs
+
+    ```
+    ExampleModel.where("boolean_column = 't'").update_all(boolean_column: 1)
+    ExampleModel.where("boolean_column = 't'").update_all(boolean_column: 0)
+    ```
+
+for all models and all boolean columns, after which the flag must be set to true
+by adding the following to your application.rb file:
+
+    ```
+    ActiveRecord::ConnectionAdapters::SQLite3Adapter.represent_boolean_as_integer = true
+    ```
```

Comment:
Thanks for working on this! I think the indentation coupled with the backticks are creating odd markdown for this part. If you click "view" on this file it shows the backticks inside a codeblock. I think if you un-indent this the extra backticks will go away. If not it's a bug in github's markdown and I'll report it to the appropriate team.

<img width="961" alt="screen shot 2017-07-11 at 2 35 42 pm" src="https://user-images.githubusercontent.com/1080678/28084361-626191ba-6646-11e7-88c5-a3f8c5a58b51.png">